### PR TITLE
[EPMRPP-40093] Swagger improvements 

### PR DESCRIPTION
**Pull request by ihar-kahadouski**

### CODE REVIEW

File: src/main/java/com/epam/reportportal/listeners/ListenerParameters.java [hunk 1]
```diff
@@ -84,6 +85,7 @@ public ListenerParameters() {
 
 	public ListenerParameters(PropertiesLoader properties) {
 		this.description = properties.getProperty(DESCRIPTION);
+		this.apiKey = properties.getProperty(API_KEY, "");
```
Comment (avarabyeu) [inline src/main/java/com/epam/reportportal/listeners/ListenerParameters.java]: if you add api key, why uuid is still there? shouldn't we remove this property

File: src/main/java/com/epam/reportportal/service/LoggingContext.java [hunk 2]
```diff
@@ -121,44 +122,37 @@ public static Completable complete() {
...
-				})
-				.observeOn(Schedulers.computation())
-				.subscribe(logFlowableResults("Logging context"));
+				}
+				return client.log(builder.build()).toFlowable();
+			}
+		}).doOnError(new Consumer<Throwable>() {
+			@Override
+			public void accept(Throwable throwable) throws Exception {
+				throwable.printStackTrace();
```
Comment (avarabyeu) [inline src/main/java/com/epam/reportportal/service/LoggingContext.java]: log?

File: src/main/java/com/epam/reportportal/listeners/ListenerParameters.java (hunk 1 above)
Comment (ihar-kahadouski) [reply src/main/java/com/epam/reportportal/listeners/ListenerParameters.java]: done

File: src/main/java/com/epam/reportportal/service/LoggingContext.java (hunk 2 above)
Comment (ihar-kahadouski) [reply src/main/java/com/epam/reportportal/service/LoggingContext.java]: done

**Pull request merged**